README.md

```diff
@@ -34,6 +34,22 @@ where:
 
 More detailed instructions are on our ["Usage"](/docs/USAGE.md) page.
 
+# Run the app using Docker
+### Setup
+1. Download and install [Docker](https://docs.docker.com/get-started/)
+1. Pull the [latest Docker image for this project](https://github.com/orgs/MobilityData/packages/container/package/gtfs-validator)
+
+### Run it
+
+To run the Docker image in a new container:
+`docker run -v /myDirectory:/theContainerDirectory -it ghcr.io/mobilitydata/gtfs-validator:v2.0.0`
```

Comment:
This doesn't seem to work for me on Windows 10 :(. I tried:

```
docker run -v /git-projects/gtfs-validator/output:/output -it ghcr.io/mobilitydata/gtfs-validator:v2.0.0
```

With this, I can execute the validator and pull GTFS files from a URL, but I can't see the GTFS zip file that's on my desktop in Docker, and I can't see the validator output on my desktop. Any idea why this wouldn't work?

Reply:
`/git-projects/gtfs-validator/output:/output` this would only give you access to what's inside `/git-projects/gtfs-validator/output` on your local machine, from `output` from the Docker container

Reply:
The `/git-projects/gtfs-validator/output` path is on my local computer. And I'm trying to use the folder `output` within the Docker container. 

My understanding is that with:

```
docker run -v /git-projects/gtfs-validator/output:/output -it ghcr.io/mobilitydata/gtfs-validator:v2.0.0
```

If I put `gtfs.zip` in my local computer folder `/git-projects/gtfs-validator/output:/output`, then within the container I should be able to do:

```
cd output
ls
```

...and it should show `gtfs.zip` within that folder.

Similarly, if I run the validator within the container and write `results.json` to the container `output` folder, then I should be able to see that file locally on my computer at `/git-projects/gtfs-validator/output:/output`.

Is the above correct, or am I understanding this wrong?

Reply:
Ah - it looks like on Windows I have to provide the drive letter too - so while the above didn't work, this does:

```
docker run -v c:/git-projects/gtfs-validator/output:/output -it ghcr.io/mobilitydata/gtfs-validator:v2.0.0
```

Does it work on your machine with a drive letter too? If so I'd recommend updating the example to include a drive letter so it works out-of-the-box for Windows users.

Reply:
Unfortunately no. Will update the docs as suggested. Thanks @barbeau.

Reply:
Done in https://github.com/MobilityData/gtfs-validator/pull/842/commits/00a9dea02096cb455958fbf4215a33a23e237722.